calicoctl/commands/node/run.go

```diff
@@ -63,17 +63,29 @@ func Run(args []string) {
 
 Options:
   -h --help                Show this screen.
-     --as=<AS_NUM>         The default AS number for this node.  If this is not
-                           specified, the node will use the global AS number
-                           (see 'calicoctl config' for details).
      --name=<NAME>         The name of the Calico node.  If this is not
                            supplied it defaults to the host name.
-     --ip=<IP>             The local management address to use.  If this is not
-                           specified, the node will attempt to auto-discover
-                           the local IP address to use - however, it is
-                           recommended to specify the required address to use.
-     --ip6=<IP6>           The local IPv6 management address to use.  If this
-                           is not specified, the node will not route IPv6.
+     --as=<AS_NUM>         Set the AS number for this node.  If omitted, the
+                           node will use the currently configured value and
+                           you can use the node resource to modify the AS
+                           number.  If there is no configured value and the
+                           the --as option is omitted, the node will inherit
+                           the global AS number (see 'calicoctl config' for
+                           details).
+     --ip=<IP>             Set the local IPv4 routing address for this node.
+                           If omitted, the node will use the currently
+                           configured value and you can use the node resource
```

Comment:
I'd reword this to "If omitted, will use the value configured on the node object."